cmd/worker/main.go

```diff
@@ -35,7 +36,28 @@ const (
 	localPkgPathFmt = "/local/%s"
 )
 
-func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blob.Bucket, resultsBucket, fileWritesBucket, imageTag string) error {
+type pkgIdentifier resultstore.PkgIdentifier
+
+type notificationMessage struct {
+	Package	pkgIdentifier
+}
+
+func publishNotification(ctx context.Context, topicNotification *pubsub.Topic, pkgDetails pkgIdentifier) error {
+	notificationMsg, err := json.Marshal(notificationMessage{pkgDetails})
+	if err != nil {
+		return fmt.Errorf("failed to create notification message for the completed analysis for a new package: %v", err)
```

Comment:
nit: we can be more concise and precise here. e.g.

```
failed to encode completion notification: %w
```

Also, we should be using "%w" instead. See https://go.dev/blog/go1.13-errors